partcad-cli/src/partcad_cli/click/commands/init.py

```
@@ -14,19 +14,19 @@
 @click.command(help="Create a new PartCAD package in the current directory")
 # TODO-95: All long options everywhere
-@click.option("-p", is_flag=True, help="Initialize this package as private")
+@click.option("-p", "--private", is_flag=True, help="Initialize this package as private")
 @click.pass_context
-def cli(ctx: click.rich_context.RichContext, p):
+def cli(ctx: click.rich_context.RichContext, private):
     # TODO-96: @alexanderilyin: Deal with noise like "PartCAD configuration file is not found"
     logging.had_errors = False
-    if not ctx.parent.params.get("p") is None:
-        if os.path.isdir(ctx.parent.params.get("p")):
+    if not ctx.parent.params.get("package") is None:
```

Ruff (0.8.2)

Test for object identity should be `is not`

Convert to `is not`

(E714)